### DISCUSSION TITLE
kubectl.plugin.zsh:18: command not found: kubectl

### DISCUSSION THREAD

**Discussion by brettcurtis** [ohmyzsh/ohmyzsh#10737] [Help] [answered]
I've been seeing a lot of bug fixes around kubectl and I'd say a few weeks back this started popping up each time I open Windows Terminal.

![image](https://user-images.githubusercontent.com/1610100/155728123-e82802f9-4449-4d53-b58a-8596b541f7db.png)

I'm not sure what's up, the plugin works. If a source my .zshrc I don't see the issue only when I start the Terminal. I also don't see it in my VScode terminal.

**Comment by carlosala** [comment]
You have to change the $PATH before sourcing `oh-my-zsh.sh`!

**Reply by brettcurtis** [reply]
Yeah I have my $PATH first in my .zshrc
![image](https://user-images.githubusercontent.com/1610100/155730029-dac4eea7-e45f-494b-8fd7-706073cf5370.png)

I think it's some ordering thing too but it's not clear what.

**Comment by carlosala** [comment, marked as answer]
This path does not look right: it doesn't appear the folder where kubectl is. The line, as far as I understand, should be:
```zsh
export PATH=$HOME/bin:$HOME/linuxbrew/.linuxbrew/lib/ruby/gems/3.0.0/bin:$HOME/linuxbrew/.linuxbrew/bin:$PATH
```
Tell me if this works for you!

**Reply by brettcurtis** [reply]
Ah yeah, that lead me to the problem:
![image](https://user-images.githubusercontent.com/1610100/155732507-c89e1bec-188d-4a42-9be9-14eaa5c0f592.png)

Adding `eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"` to setup my brew environment prior to loading plugins fixed me up.

Thanks for the second set of eyes!
